policyengine_us/parameters/simulation/de/branch_to_determine_if_refundable_eitc.yaml

```
@@ -0,0 +1,6 @@
+description: PolicyEngine branches to determine Delaware refundability if this is true; otherwise it is not refundbale.
```

"Refundability" refers to the policy choice of whether a credit is refundable. Here we're interested in the filer's choice.
```suggestion
description: PolicyEngine branches to determine whether filers claim the refundable over non-refundable Delaware EITC, as they can choose depending on which minimizes tax liability.
```